src/rard/templates/research/add_work_link.html

```diff
@@ -9,25 +14,37 @@
 
 {% block inner %}
 
-
-    {# Use a GET form to select the work and antiquarian #}
+    {# antiquarian form #}
     <form novalidate enctype="multipart/form-data" autocomplete='off' action='{{ request.path }}' class="form work-form" method='GET'>
         {% bootstrap_field form.antiquarian field_class='submit-on-change' %}
+        <span class="toggle-definite">
+        {% bootstrap_field form.definite_antiquarian %}</span>
+
     </form>
 
+    {# work form #}
     <form novalidate enctype="multipart/form-data" autocomplete='off' action='{{ request.path }}' class="form work-form" method='GET'>
         {% bootstrap_field form.work field_class='submit-on-change' %}
+        <span class="toggle-definite">
+        {% bootstrap_field form.definite_work %}</span>
         <input type='hidden' name='antiquarian' value='{{ antiquarian.pk }}' />
-    </form>
+        <input type='hidden' name='definite_antiquarian' {% if definite_antiquarian %} value={{ definite_antiquarian }} {% endif %} />
```

Comment:
If I select definite antiquarian after first page reload, then select a work, the definite antiquarian box is unchecked when the page reloads again. Would it be possible to add some JS to update this input value when user changes definite checkboxes?

Reply:
We do have that, but when you select an antiquarian it then submits the form, so you're doing it in the limbo of the page refresh. I'll check what's going on as it used to function
Should be lines 57-72 for the script